Promact.Oauth.Server/src/Promact.Oauth.Server/Startup.cs

```diff
@@ -27,9 +27,10 @@ namespace Promact.Oauth.Server
     public class Startup
     {
         private ILoggerFactory _loggerFactory { get; }
-
+        private IHostingEnvironment CurrentEnvironment { get; set; }
```

Comment:
Yes. But ConfigureServices is where we have done the resolving of implementations of IEmailSender.cs
  if (CurrentEnvironment.IsDevelopment())
                services.AddTransient<IEmailSender, AuthMessageSender>();
            else if (CurrentEnvironment.IsProduction())
                services.AddTransient<IEmailSender, SendGridEmailSender>();